### PR TITLE
Add user as sudoer

### PR DESCRIPTION
**Pull request by walteraa**
This PR is referring to the issue #3  , now is possible to add a user with sudo permission. 

This feature is using sudoers file to give sudo permission for an user always the sudo property is true.

### CODE REVIEW

File: lib/sshpm/tasks/add_user.rb [hunk 1]
```diff
@@ -49,6 +49,12 @@ def run_on(host)
           ssh.exec! "chmod 600 #{auth_keys_file}"
           ssh.exec! "chown -R #{name}:#{name} #{ssh_dir}"
         end
+
+        if sudo 
+          ssh.exec! "echo \"#{name} ALL=(ALL) NOPASSWD:ALL\" >> /etc/sudoers"
+        end
+
```
Comment (terciodemelo) [inline lib/sshpm/tasks/add_user.rb]: Would you remove these extra blank lines, please?

File: spec/sshpm_spec.rb [hunk 2]
```diff
@@ -146,6 +146,165 @@
...
+
+          it "user can use sudo on all test servers" do
+                opts = {
+                  password: @user[:password],
+                  port: @port,
+                  paranoid: false
+                }
+                
+                Net::SSH.start('localhost', @user[:username], opts) do |ssh|
+
```
Comment (terciodemelo) [inline spec/sshpm_spec.rb]: Would you remove the extra blank line, please?

File: spec/sshpm_spec.rb [hunk 3]
```diff
@@ -146,6 +146,165 @@
...
+          it "user cannot use sudo on all test servers" do
+                opts = {
+                  password: @user[:password],
+                  port: @port,
+                  paranoid: false
+                }
+                
+                Net::SSH.start('localhost', @user[:username], opts) do |ssh|
+
```
Comment (terciodemelo) [inline spec/sshpm_spec.rb]: Would you remove the extra blank line, please?

File: spec/sshpm_spec.rb [hunk 4]
```diff
@@ -146,6 +146,165 @@
...
+                public_key user[:public_key]
+                sudo user[:sudo]
+              end
+            end
+          end
+
+          it "user can use sudo on all test servers" do                    
+                opts = SSHPM::Tests.ssh_identity_options port: @port, key_data: [@rsa_key.private_key]
+                Net::SSH.start('localhost', @user[:username], opts) do |ssh|
+
```
Comment (terciodemelo) [inline spec/sshpm_spec.rb]: Would you remove the extra blank line, please?

File: spec/sshpm_spec.rb [hunk 5]
```diff
@@ -146,6 +146,165 @@
...
+                name user[:username]
+                public_key user[:public_key]
+              end
+            end
+          end
+
+          it "user cannot use sudo on all test servers" do
+                opts = SSHPM::Tests.ssh_identity_options port: @port, key_data: [@rsa_key.private_key]
+                Net::SSH.start('localhost', @user[:username], opts) do |ssh|
+
```
Comment (terciodemelo) [inline spec/sshpm_spec.rb]: Would you remove the extra blank line, please?

File: spec/sshpm_spec.rb [hunk 6]
```diff
@@ -146,6 +146,165 @@
...
+          it "user cannot use sudo on all test servers" do
+                opts = SSHPM::Tests.ssh_identity_options port: @port, key_data: [@rsa_key.private_key]
+                Net::SSH.start('localhost', @user[:username], opts) do |ssh|
+                  
+                  output = ssh.exec!("sudo --prompt=\"\" ls > /dev/null")
+                  expect(output).to_not be_empty
+
+                end
+
```
Comment (terciodemelo) [inline spec/sshpm_spec.rb]: Would you remove the extra blank lines, please?

**Comment by terciodemelo** [comment]
Very good work @walteraa!
I think you just need to fix some styling in the files
I'll add a check to github so that this can be verified automatically on commit

**Comment by walteraa** [comment]
Done, @terciodemelo!

**Comment by terciodemelo** [comment]
This PR solves #3

**Pull request merged**